lib/console-manager.js

```diff
@@ -16,6 +16,7 @@ export default class ConsoleManager {
 
 	// Toggle console panel
 	toggle() {
+		alert("hola")
```

Comment:
I think this was for debug  purposes and is no longer needed 😅

Reply:
looool! Yes :/

Reply:
fixed!